Add comparison to exact optimal function value using `CplexOptimizer` (#27)
qiskit-community/prototype-qrao: requirements.txt

qiskit-terra>=0.19.2
qiskit-aer>=0.10.3
qiskit-ibmq-provider>=0.18.3

> qiskit-optimization[cplex]>=0.3.0

Reviewer: ```suggestion
qiskit-optimization>=0.3.0
```

Reverting this change since it is actually [redundant](https://github.com/Qiskit/qiskit-optimization/blob/ea045123fdaeaed3789731ef01160fca0f8db4a1/setup.py#L72) with having specified `cplex` explicitly later in the file.  That way is preferred, because it at least allows us to specify a minimum version.

matplotlib>=3.0.0

> # The following line isn't strictly required by qrao if it is used as
> # a library, but it's required for one of the included Jupyter
> # notebooks, so we include it in this prototype's requirements for
> # convenience.
> cplex>=12.10

Reviewer: ```suggestion
# The following lines aren't strictly required by qrao if it is used
# as a library, but they are required for the included Jupyter
# notebooks, so we include them in this prototype's requirements for
# convenience.
cplex>=12.10
```
